surface_potential_analysis/state_vector/plot_value_list.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, TypeVar
+
+from matplotlib import pyplot as plt
+
+from surface_potential_analysis.basis.time_basis_like import EvenlySpacedTimeBasis
+
+if TYPE_CHECKING:
+    from surface_potential_analysis.state_vector.eigenstate_collection import ValueList
+
+_AX0Inv = TypeVar("_AX0Inv", bound=EvenlySpacedTimeBasis[Any, Any, Any])
+
+
+def plot_value_list_abs_against_time(
```

Comment:
Following the example from elsewhere, it should take 
```
   *,
    ax: Axes | None = None,
    measure: Measure = "abs",
    scale: Scale = "linear",
```
as keyword arguments

would be good if this could take a ValueList[_Ax0Inv] rather than times explicitly. 

Copy code from plot_data_1d_k to make sure you get correct margins etc. I will refactor this in a bit to add a plot_data_1d which is generic over k, x, time etc so don't worry about importing anything which starts with an unsderscore form this file...